recipes-ni/ni-base-system-image-tests/files/fs_permissions_known.py

```diff
@@ -27,13 +27,161 @@ def known_permissions_tree():
+                            'faddr2line': system_file_exec,
+                            'gcc-ld': system_file_exec,
+                            'genksyms': {
+                                'genksyms': system_file_exec
+                            },
+                            'get_dvb_firmware': system_file_exec,
+                            'gfp-translate': system_file_exec,
+                            'jobserver-exec': system_file_exec,
+                            'kallsyms': system_file_exec,
+                            'kconfig': {
+                                'conf': system_file_exec
+                            },
+                            'kernel-doc': system_file_exec,
+                            'Lindent': system_file_exec,
+                            'makelst': system_file_exec,
+                            'mkcompile_h': system_file_exec,
+                            'mksysmap': system_file_exec,
+                            'mod': {
+                                'mk_elfconfig': system_file_exec,
+                                'modpost': system_file_exec
+                            },
+                            'objdiff': system_file_exec,
+                            'package': {
+                                '*': system_file_exec,
+                                'snapcraft.template': system_file
+                            },
+                            'patch-kernel': system_file_exec,
+                            'prune-kernel': system_file_exec,
+                            'remove-stale-files': system_file_exec,
+                            'selinux': {
+                                'genheaders': {
+                                    'genheaders': system_file_exec,
+                                },
+                                'mdp': {
+                                    'mdp': system_file_exec
+                                }
+                            },
+                            'setlocalversion': system_file_exec,
+                            'show_delta': system_file_exec,
+                            'sorttable': system_file_exec,
+                            'spdxcheck.py': system_file_exec,
+                            'spdxcheck-test.sh': system_file,
+                            'sphinx-pre-install': system_file_exec,
+                            'stackdelta': system_file_exec,
+                            'stackusage': system_file_exec,
+                            'tracing': {
+                                'draw_functrace.py': system_file_exec
+                            },
+                            'ver_linux': system_file_exec,
+                        },
+                        'tools': {
+                            'lib': {
+                                'lockdep': {
+                                    'lockdep': system_file_exec,
+                                    'run_tests.sh': system_file_exec
```

Comment:
Shouldn't this and other .sh entries by covered already by the **/*.sh rule above?

Reply:
No. That's for `/lib/modules/$(uname -r)/build/scripts/**/*.sh`
This script is in `/lib/modules/$(uname -r)/build/tools/lib/lockdep/`

I suppose I could move the `**/*.sh` up to the build directory, though it would only add this script and `/lib/modules/$(uname -r)/build/objtool/sync-check.sh`

Reply:
I would say it's worth moving up to the build directory. Pays a small dividend now and maybe more in the future.